build.gradle

```diff
@@ -12,7 +12,7 @@ apply plugin: 'signing'
 
 allprojects {
     group = 'com.cloudant'
-    version = '0.15.6-SNAPSHOT'
+    version = '1.0.0-SNAPSHOT'
```

Comment:
I'll split into a separate commit before merging, so I only need to make one force push